google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/HeaderTracerStreamingCallable.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.stub.metrics;
+
+import com.google.api.core.InternalApi;
+import com.google.api.gax.grpc.GrpcResponseMetadata;
+import com.google.api.gax.rpc.ApiCallContext;
+import com.google.api.gax.rpc.ResponseObserver;
+import com.google.api.gax.rpc.ServerStreamingCallable;
+import com.google.api.gax.rpc.StreamController;
+import io.grpc.Metadata;
+import javax.annotation.Nonnull;
+
+/**
+ * Record GFE metrics.
+ *
+ * <p>This class is considered an internal implementation detail and not meant to be used by
+ * applications.
+ */
+@InternalApi
```

Comment:
Please add a comment as to what this does. It will be helpful for future maintainers..Something like:

This callable will inject GrpcResponseMetadata into the ApiCallOptions. Upon rpc completion, gax will inject grpc trailers into this class. The trailers will then be processed by HeaderTracer

Reply:
Also, can you add the new internal api classes to this exclusion list:
https://github.com/googleapis/java-bigtable/blob/master/pom.xml#L252

Reply:
Good to know! In this case my classes were added under metrics so I think it's covered by: https://github.com/googleapis/java-bigtable/blob/master/pom.xml#L285